src/http-api/resources/index.js

```diff
@@ -5,6 +5,7 @@ exports.id = require('./id')
 exports.bootstrap = require('./bootstrap')
 exports.repo = require('./repo')
 exports.object = require('./object')
+//exports.pin = require('./pin')
```

Comment:
I haven't tried to update the http tests yet so I left it commented. Several other tests are also temporarily commented as I didn't see a better way to get just the pin tests to run without the rest.